stan/math/rev/core/init_chainablestack_tbb.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef STAN_MATH_REV_CORE_INIT_CHAINABLESTACK_TBB_HPP
+#define STAN_MATH_REV_CORE_INIT_CHAINABLESTACK_TBB_HPP
+
+// TODO(SW): remove STAN_THREADS guard once Intel TBB is fully
+// mandatory
+#ifdef STAN_THREADS
+
+#include <stan/math/rev/core/chainablestack.hpp>
+
+#include <tbb/task_scheduler_observer.h>
+
+#include <mutex>
+#include <iostream>
+#include <unordered_map>
+#include <thread>
+#include <utility>
+#include <tuple>
+
+namespace stan {
+namespace math {
+
+static std::mutex cout_mutex;
```

Comment:
For stuff like this I find it helpful to have a `// FIXME(DELETE)` somewhere just so I remember to delete this stuff